allensdk/brain_observatory/behavior/session_apis/data_transforms/behavior_data_transforms.py

```diff
@@ -227,6 +229,16 @@ def get_stimulus_templates(self) -> Optional[StimulusTemplate]:
         pkl = self._behavior_stimulus_file()
         return get_stimulus_templates(pkl=pkl)
 
+    def get_monitor_delay(self) -> float:
+        """
+        Return monitor delay for behavior only sessions
+        (in seconds)
+        """
+        # This is the median estimate across all rigs
+        # as discussed in
+        # https://app.zenhub.com/workspaces/allensdk-10-5c17f74db59cfb36f158db8c/issues/alleninstitute/allensdk/1318
```

Comment:
this may be a more persistent link:
https://github.com/AllenInstitute/AllenSDK/issues/1318

Reply:
done (I also linked to issue 1916, since that is where the final lookup table was published)

Reply:
wait, I see you did the change down in 
`behavior_ophys_data_transforms.py`
but here in
`behavior_data_transforms.py`
I srtill see the zenhub link.

Reply:
eek. Good catch.